### ISSUE TITLE
invalid literal for int() with base 10: '' on forms/add

### ISSUE THREAD

**Issue by TerraHub** [stephenmcd/mezzanine#1232]
This is a clean install of mezzanine 3.1.10 on Django 1.7.  I get this whenever using the admin for forms/add. This happens with a fresh local install running with DEBUG=True as well as installs on EC2 serving static files from S3.  Looking at the JS console, I do see one javascript error on forms/add: 

3254: Cannot read property 'focus' of null

3254: <script type="text/javascript">document.getElementById("").focus();</script>

**Comment by stephenmcd** [comment]
Can you reproduce the error on the demo site?

http://mezzanine.jupo.org/en/admin/

The username and password is demo/demo

**Comment by TerraHub** [comment]
Hmmm. No, I cannot. Is the settings file for the demo site available, and also possibly the output of a pip freeze? I've narrowed down the issue to the DynamicTabularInline that allows you to add new form fields dynamically.  I was able to create a new form with no fields on my site, and I've been able to save a form after modifying existing fields (by creating them in the shell first). I still cannot, even in a clean install on my own machine, get past this error.  If I find something that can be patched, I'll commit the patch back or at least document what I had to change.  Thanks for looking into this!
